examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -85,6 +85,22 @@ void ModelCommand::ClearICDEntry(const chip::ScopedNodeId & nodeId)
     }
 }
 
+void ModelCommand::AddICDEntry(chip::app::ICDClientInfo & clientInfo, chip::ByteSpan key)
```

Comment:
Why is that first arg non-const?

Reply:
sICDClientStorage.SetKey is happening inside this function,
just rename this function to StoreICDEntryWithKey to reflect this. thanks